### ISSUE TITLE
Installation fails "unable to update url base from redirection"

### ISSUE THREAD

**Issue by pv2b** [okkur/syna-start#21]
When attempting to follow the install directions as outlined at https://github.com/okkur/syna-start, this is what I get:

```
PS C:\hugo\sites> git clone https://git.okkur.org/syna-start myproject
Cloning into 'myproject'...
warning: redirecting to https://github.com/okkur/syna-start.git/
remote: Enumerating objects: 19, done.
remote: Counting objects: 100% (19/19), done.
remote: Compressing objects: 100% (16/16), done.
remote: Total 367 (delta 7), reused 9 (delta 2), pack-reused 348
Receiving objects: 100% (367/367), 947.13 KiB | 2.34 MiB/s, done.
Resolving deltas: 100% (144/144), done.
PS C:\hugo\sites> cd myproject
PS C:\hugo\sites\myproject> git submodule init
Submodule 'themes/syna' (https://git.okkur.org/okkur/syna) registered for path 'themes/syna'
PS C:\hugo\sites\myproject> git submodule update
Cloning into 'C:/Hugo/Sites/myproject/themes/syna'...
fatal: unable to update url base from redirection:
  asked for: https://git.okkur.org/okkur/syna/info/refs?service=git-upload-pack
   redirect: https://about.okkur.org/
fatal: clone of 'https://git.okkur.org/okkur/syna' into submodule path 'C:/Hugo/Sites/myproject/themes/syna' failed
Failed to clone 'themes/syna'. Retry scheduled
Cloning into 'C:/Hugo/Sites/myproject/themes/syna'...
fatal: unable to update url base from redirection:
  asked for: https://git.okkur.org/okkur/syna/info/refs?service=git-upload-pack
   redirect: https://about.okkur.org/
fatal: clone of 'https://git.okkur.org/okkur/syna' into submodule path 'C:/Hugo/Sites/myproject/themes/syna' failed
Failed to clone 'themes/syna' a second time, aborting
PS C:\hugo\sites\myproject>
```

It would appear that this is a misconfiguration of git.okkur.org, making it redirect to https://about.okkur.org/ which is not a repo.

**Comment by pv2b** [comment]
I've identified a workaround, which is to change the URL in .gitmodules to https://github.com/okkur/syna

After that, you need to run git submodule sync.

Not sure if you want to change this in code to hard-point to github or if you'd rather fix your redirects.

**Comment by stp-ip** [comment]
We are using a vanity url for git and it wasn't correctly migrated. This should be fixed with #22

**Comment by stp-ip** [comment]
Thanks for catching this.
